cmd/krew/cmd/util_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"sigs.k8s.io/krew/internal/testutil"
+	"sigs.k8s.io/krew/pkg/constants"
+	"sigs.k8s.io/krew/pkg/index"
+)
+
+func Test_isDefaultIndex(t *testing.T) {
+	if !isDefaultIndex("") {
+		t.Error("empty string must indicate default index")
+	}
+	if !isDefaultIndex("default") { // nb: intentionally not using the const to ensure compatibility
+		t.Error("default index must indicate default index")
+	}
+	if isDefaultIndex("foo") { // nb: intentionally not using the const to ensure compatibility
```

Comment:
```suggestion
	if isDefaultIndex("foo") {
```

Reply:
Also just for my own clarification, what does `nb` stand for?

Reply:
nota bene